Arduino_BHY2/src/BLEHandler.h

```diff
@@ -10,14 +10,37 @@
 #define BHY2_ENABLE_BLE_BATCH 1
 const uint8_t BLE_SENSOR_EVT_BATCH_CNT_MAX = (244 / (SENSOR_DATA_FIXED_LENGTH + 2))/10*10;
 
+/**
+ * @brief Class for handling BLE communication for both sensor data transfer and DFU
+ * 
+ */
 class BLEHandler {
 public:
   BLEHandler();
   virtual ~BLEHandler();
 
+  /**
+   * @brief Set advertised local name and initialise BLE advertising for DFU and sensor data transfer
+   * 
+   * @return true   Successful initialisation of BLE device
+   * @return false  Failure in initialisation of BLE device
+   */
   bool begin();
+  /**
+   * @brief Poll data over bluetooth and read data to FIFO buffer.
```

Comment:
```suggestion
   * @brief Poll data over Bluetooth and read data to FIFO buffer.
```

"Bluetooth" is a proper noun.

Reference: https://en.wikipedia.org/wiki/Bluetooth